appveyor.yml

```diff
@@ -7,3 +9,12 @@ test_script:
 
 build_script:
 - dotnet build ./src/ -v m -c Release
+
+build:
+  publish_nuget: true
+
+deploy:
+  provider: NuGet
+  api_key: oy2n3sxah4nvfc4lfphquuynozc37bfujjfhjy3bvhmt64
```

Comment:
Never expose your API Key like that! 
Use the approach from [Composite project](https://github.com/vkamiansky/composite/blob/master/appveyor.yml)
Only publish releases on master tagged with version number.
Reuse secure API key from Composite. We'll publish it under vkamiansky. We might want to remove this one https://www.nuget.org/packages/FlsResults/.